humanguide/general-concepts/index.rst

```diff
@@ -0,0 +1,12 @@
+General concepts
+================
+
+.. toctree::
+    :maxdepth: 1
+    :name: general-concepts
+
+    01-system-palette
+    02-units
+    03-typography
+    04-app-icon-design
```

Comment:
Pages will often be reordered or moved entirely. It is not a good idea to put their order in their name.